generated/serialization.h

```diff
@@ -0,0 +1,39 @@
+#ifndef DTOGEN_SERIALIZATION_H
+#define DTOGEN_SERIALIZATION_H
+
+#include "nlohmann/json.hpp"
```

Comment:
As you can see, the code generation places the header-only json.hpp to a well-known path. It's not difficult to add custom options to the generator, so we can configure this behavior.
I guess ultimately we want to use Conan instead.